dts/xtensa/intel/intel_adsp_cavs18.dtsi

```diff
@@ -79,6 +79,7 @@
 			reg = <0x71a08 0x8>;
 			memory = <&sram0>;
 			read-only;
+			status = "okay";
```

Comment:
Something is wrong, I think.

This change doesn't actually do anything as far as I can tell.

Note that a missing status property is treated as if it were "okay". So you would have to explicitly change "disabled" to "okay" to have an affect. Adding a missing "okay" does nothing.

Reply:
> Note that a missing status property is treated as if it were "okay".

https://docs.zephyrproject.org/latest/build/dts/intro.html#important-properties

Reply:
> Something is wrong, I think.

it is okay, I just wanted to have them all enabled in a consistent way so it does not appear as if some are enabled and others are not. I could remove this alltogather and use the default for simplicity I guess, but the way it is now should be good as well.

Reply:
agree, let me remove that and use defaults instead.